File: ios/Sources/MeasureSDK/Swift/Tracing/SpanStatus.swift
//
//  SpanStatus.swift
//  Measure
//
//  Created by Adwin Ross on 08/04/25.
//

import Foundation

/// Specifies the status of the operation for which the span has been created.
public enum SpanStatus: Int64, Codable {
    case unset = 0
    case ok = 1
    case error = 2
}

File: kmp/measure-kmp/src/commonMain/kotlin/sh/measure/kmp/tracing/SpanStatus.kt
package sh.measure.kmp.tracing

/**
 * Specifies the status of the operation for which the span has been created.
 */
enum class SpanStatus {
    /**
     * Default value for all spans.
     */
    Unset,

    /**
     * The operation completed successfully.
     */
    Ok,

    /**
     * The operation ended in a failure.
     */
    Error,
}
